src/ray/core_worker/core_worker.cc

```diff
@@ -1033,7 +1154,8 @@ void CoreWorker::RegisterToGcs(int64_t worker_launch_time_ms,
   worker_data->mutable_worker_address()->set_port(rpc_address_.port());
   worker_data->mutable_worker_address()->set_worker_id(worker_id.Binary());
   worker_data->set_worker_type(options_.worker_type);
-  worker_data->mutable_worker_info()->insert(worker_info.begin(), worker_info.end());
+  worker_data->mutable_worker_info()->insert(std::make_move_iterator(worker_info.begin()),
+                                             std::make_move_iterator(worker_info.end()));
```

Comment:
hmm protobuf map doesn't have a reserve, but i feel like it should if it's implemented similarly to unordered or even flatmap...

Reply:
Using move iterator is the general practice; for example, if protobuf implements reservation, we don't need to change any code.